packages/webapp/src/components/PrivateRoute.tsx

```diff
@@ -71,7 +71,7 @@ export const PrivateRoute: React.FC = () => {
         }
     }, [user, identify]);
 
-    if (loading || !ready) {
+    if (loading || !ready || loadingUser) {
```

Comment:
what's the difference between loading and loadingUser

Reply:
let me rename the other loading 👌🏻